src/main/webapp/app/modules/managerSurveysDashboard/managerSurveysDashboard.tsx

```diff
@@ -89,7 +89,7 @@ const ManagerSurveysDashboard = () => {
               <BsGear className={'icon'} />
               Modify
             </Dropdown.Item>
-            <Dropdown.Item>
+            <Dropdown.Item as={Link} to="/SurveyStatusView">
```

Comment:
you declared a const `surveyPath` but you don't use it here. Decide if you want to use variable or not :-)